crates/polars-ops/src/series/ops/search_sorted.rs

```diff
@@ -15,6 +16,14 @@ pub fn search_sorted(
         polars_bail!(InvalidOperation: "'search_sorted' is not supported on dtype: {}", s.dtype())
     }
 
+    let (s, search_values) = if s.dtype().is_array() || s.dtype().is_list() {
```

Comment:
Shouldn't this be `s.dtype().is_nested()` as to also take into account structs?

Reply:
Ah, apparently this is because Ritchie added nested supported a few days ago. So going to have to make some changes.